e2e/basic/basic_test.go

```diff
@@ -1,53 +1,60 @@
 package basic
 
 import (
-	"testing"
+	"context"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 
-	"github.com/celestiaorg/knuu/pkg/knuu"
+	"github.com/celestiaorg/knuu/pkg/instance"
 )
 
-func TestBasic(t *testing.T) {
-	t.Parallel()
+const (
+	TestImage = "alpine:latest"
```

Comment:
two points about this one:
1. This does not have to be starting with a Capital letter as we do not really need to export this constant.
2. It can have some more specific name as it is accessible package wise so other basic tests also can see this.